src/SymmetricFunctions/wreath_macdonald_polynomials.jl

```diff
@@ -0,0 +1,357 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+import Chevie: complex_reflection_group, charinfo, CharTable, CyclotomicNumbers
```

Comment:
This won't work because `Chevie` is not listed as a dependency in `Project.toml`. I.e. Oscar will not even precompile like this...

However overall I don't think adding `Chevie` as a dependency will be acceptable. But as @ulthiel already suggested there may be alternatives...

Reply:
For the meantime (aka to get this PR going), IMO it is a solution to add `Chevie` as a dependency in `Project.toml`. That way, one could work on getting CI happy, cleaning up the code a bit, adding tests and docs etc, and in general have an easier time working against bitrotting. 
Of course, this dependency would then need to be removed again (before merging) and replaced with the alternative.

Reply:
The right person to talk to about character tables is @ThomasBreuer.